layer1/chain/src/chain.mligo

```diff
@@ -0,0 +1,77 @@
+#import "../../stdlib_ext/src/stdlibext.mligo" "Stdlib"
+
+type hash = bytes
+type index = nat
+type batch = 
+    {  parent : index
+    ;  level : nat
+    ;  hash : hash
+    ;  proposer : address
+    }
+type chain = 
+    {   max_index : nat
```

Comment:
Nitpicking : why _"max"_ in _"max_index"_ ? I think index is sufficient

Reply:
each batch has an index, `max_index` is the highest that has been attributed